include/materials/norsand.h

```diff
@@ -69,13 +69,11 @@ class NorSand : public Material<Tdim> {
   void compute_plastic_tensor(const Vector6d& stress,
                               mpm::dense_map* state_vars);
 
-  //! Compute stress invariants (p, q, j2, j3, lode_angle and M_theta)
+  //! Compute stress invariants (p, q, lode_angle and M_theta)
```

Comment:
Could you please update the doxygen input parameters and return value

Reply:
Ok, please check whether it is as expected